### ISSUE TITLE
s02_depend

### ISSUE THREAD

**Issue by jessfroe** [lzim/teampsd#556]
# s02 CHECKLIST

## 1. Link to the document and add relevant team members to the card.
- [x] Add the link the current draft of the documentation for the issue:

Session 2: https://bcove.video/333AYpE

## 2. Link to issues that must be edited in the document in the **issue_tracker** and **feature_tracker.** _(use the # for the link to the issue)_
- [ ] Issue #429 
- [ ] Issue #451 
- [ ] Issue #419 
- [ ] Issue #440  

## 3. Team Review
- [ ] Fidelity - check with Lindsey
- [ ] Data UI - check quant_workgroup
- [ ] Sim UI - check with model_workgroup and/or sim_workgroup

## 4. Review and Open Pull Requests _(use the # for the link to the pull request)_
- [ ] Check the draft for typos
- [ ] Check or open a pull request for the document edits

## 5. Review and/or Update Graphics
- [ ] Screenshots
- [ ] Gifs
- [ ] Logos/Icons
- [ ] Links

## 6. Dependencies
- [ ] Make sure that any edits you make that need to be added to **depend_product** are tracked (e.g., _MTL Video_)

## 7. Close Pull Requests
- [ ] Request review and/or close open pull requests for the document

**Comment by staceypark** [comment]
Will need to check videos for data UI changes in #469 #390

**Comment by jessfroe** [comment]
@dlkibbe @branscombj 
Do you have an ETA for this video review?

**Comment by dlkibbe** [comment]
Tomorrow @jessfroe

**Comment by jessfroe** [comment]
@dlkibbe 
We only have three sessions left for video review - WOO! s02, s11, and s12.
Do you think we could have these edits completed by the end of week? It would be amazing to send all of the edits to Ann before the Thanksgiving holiday 🦃

**Comment by dlkibbe** [comment]
*Session 2 Video Review –Check Patient Data and Team Trends* LZ & TR

**This is only the 2nd time people are going to mtl.how and the first time people are logging into the data user interface. Both verbal and visual cues are needed to help the learner understand the framework and process for MTL participation. This is why I've suggested so many additions to this video.**

Add Pop up 0:22 seconds:  Add “Open the learner see guide at mtl.how”
Add Pop-up 0:36 “Done” = What we did last or in between sessions
Add Pop-up 0:49  “Do” = What we will do today
Add pop-up 0:56  “mtl.how/data is Data User Interface or Data UI”
Add pop-up 1:44 “Log in to mtl.how/data in Explorer” 
Add pop-up 2:01 “BISL = Business Intelligence Service Line” (LZ is saying this)
Add pop-up 2:15 Go to mtl.how data; enter email & PIV pass code”
Add pop-up at 2:32 “mtl.how/data contains protected health information (PHI)”
Add pop-up at 4:25 Explore MTL Data UI using vertical & horizontal scroll bars
5:29 – maybe add a pop-up: At the moment LZ is moving her cursor up to refresh the data, the camera goes to her and TR on screen vs. showing the computer screen. She is saying at “navigate up here and choose data...” and then the camera goes back to screen but the learner didn’t see her initial click. So may want to add a pop-up at 5:29 that says “Click “DATA” above chart, Click “Refresh All Connections”
Add pop-up 6:35 “Tabs with VIZ = visualization” 
Add pop-up 7:57 ”Viz Tabs = Diagnoses, Health Factors, Encounters, Measures”
Add pop-up 8:45 “Data Tabs = feature patient-level data”
9:14 May want to leave the “We’ve removed all PHI…” pop-up up longer than 5 seconds; it takes a minute to read. Edit needed – last sentence in this pop-up needs a period.
Add pop-up at 10:48 “MTL Data UI has “hyperlocal data” – by patient, by provider, by clinic”
Add pop-up at 12:40 “Click Edit Workbook, Edit in Excel” to open your team data file”
Add pop-up at 14:50 “Use MTL Data UI to create a personalized team data file”
16:15-16:34 - There are 3 pop-ups in pretty quick succession; The pop-up at 16:27 which only appears for a couple of seconds “Add All gets all clinics based on filters, not those highlighted” should have small letters and quotes around the “add all” so the learner knows this represents cell c6 on the screen
Add pop-up 17:15 “Once clinics are selected, click “Get Patient Level Data.” Generating a data file may take up to 15 minutes!”
Add pop-up 18:29 “Review team trends on viz tabs”
Add pop-up at 20:00 “Pause, Use learner see guide to walk through MTL Data UI tabs”
Add pop-up at 22:47 “Review patient data on data tabs”
Add pop-up at 23:55 “Patients engaged in specific service = Health Factors tab”
Add pop-up 26:45 “MTL Data UI = mtl.how/data”
Add pop-up at 28:15  “For MTL Help - email mtl.help@va.gov”
